src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ActiveDirectorySite.cs

```diff
@@ -1302,16 +1302,18 @@ private void GetDomains()
 
                 IntPtr info = (IntPtr)0;
                 // call DsReplicaSyncAllW
-                IntPtr functionPtr = global::Interop.Kernel32.GetProcAddress(DirectoryContext.ADHandle, "DsListDomainsInSiteW");
-                if (functionPtr == (IntPtr)0)
+                var dsListDomainsInSiteW = (delegate* unmanaged<IntPtr, char*, IntPtr*, int>)global::Interop.Kernel32.GetProcAddress(DirectoryContext.ADHandle, "DsListDomainsInSiteW");
```

Comment:
This library targets netstandard2.0. The default unmanaged calling convention is not supported for netstandard2.0. I think these needs to be explicit StdCall to make it work for netstandard2.0.

Reply:
It looks like the `netstandard2.0` implementation is a PlatformNotSupported-stubbed out implementation, so it doesn't compile this code.

Reply:
> It looks like the `netstandard2.0` implementation is a PlatformNotSupported-stubbed out implementation, so it doesn't compile this code.

That is good. I had thought that targeting `netstandard2.0` without stating the calling convention would result in a compiler error. Right?

Reply:
Yep. The feature is conditionally available based on a RuntimeFeature field/property.